open_mastr/mastr.py

```diff
@@ -76,7 +76,9 @@ class Mastr:
 
```

Comment:
A documentation of this new parameter is missing. @Simon-Will Can you add it here? It will than automatically be displayed in most code editors and on our documentation page.